src/themes/shared/static-styles.ts

```diff
@@ -4,11 +4,21 @@
  * ! Avoid adding anything to this export, unless absolutely necessary.
  * ! Rather, refactor these definitions into component-specific styles.
  */
+
+const focusableSelectors = [
+  ":focus",
+  ".ui-dropdown__container",
+  ".ui-dropdown__item",
+];
+
 export const staticStyles = [
-  `html[data-whatinput="keyboard"] :focus::after, html[data-whatinput="keyboard"] :focus::before {
+  `${focusableSelectors
+    .map(
+      (selector) =>
+        `html[data-whatinput="keyboard"] ${selector}::before, html[data-whatinput="keyboard"] ${selector}::after`
+    )
+    .join(", ")} {
     border-width: 2px !important;
-  }`,
-  `html[data-whatinput="keyboard"] :focus::after, html[data-whatinput="keyboard"] :focus::before {
     border-radius: 4px !important;
```

Comment:
if it's common and it's a part of Shell 2.0 styles, should be declared there and overwrited default borderRadius across the project?

Reply:
`themes/shared` is where many of the Shell 2.0 styles are declared, so I think this is the best place for this.